includes/class-migration-checks.php

```diff
@@ -31,7 +31,7 @@ public static function run() {
 	 * Register migration checks.
 	 */
 	public static function register() {
-		add_filter( 'bluehost_site_migrator_can_migrate', array( __CLASS__, 'can_mysqldump' ), 5 );
+		//add_filter( 'bluehost_site_migrator_can_migrate', array( __CLASS__, 'can_mysqldump' ), 5 );
```

Comment:
```suggestion
```
dont just comment out code. this is fine in development, but for the pr if you are removing code remove it